docker/GPU/Dockerfile

```diff
@@ -34,18 +35,20 @@ RUN curl -fsSL -v -o ~/miniconda.sh -O  https://repo.anaconda.com/miniconda/Mini
     /opt/conda/bin/conda install -y python=${PYTHON_VERSION} conda-build pyyaml numpy ipython cython typing typing_extensions mkl mkl-include ninja && \
     /opt/conda/bin/conda clean -ya
 
-### pytorch
-RUN /opt/conda/bin/conda install pytorch==${TORCH_VERSION} torchvision==${TORCHVISION_VERSION} cudatoolkit=${CUDA} -c pytorch
-ENV PATH /opt/conda/bin:$PATH
+### install pytorch openmim
+RUN /opt/conda/bin/conda install pytorch==${TORCH_VERSION} torchvision==${TORCHVISION_VERSION} cudatoolkit=${CUDA} -c pytorch -c conda-forge -y \
```

Comment:
could try to install with pip

Reply:
Now it can install by conda.
What is the relationship between mmcv or mmcv-full and mmdeploy?
Why mmdeploy1.x needs to rely on its compiled mmcv2.x to use

Reply:
Pure installation of mmdeploy only needs mmcv-lite. But if you want to deploy model of downstream repos such as mmdet, if they depend on mmcv-2.x, then, you need to install mmcv-2.x.

Reply:
@triple-Mu could you try with pip?


> could try to install with pip
> 
> ```
> /opt/conda/bin/pip install torch==1.9.0+cu111 torchvision==0.10.0+cu111 -f https://download.pytorch.org/whl/torch_stable.html
> ```